editor.planx.uk/src/pages/FlowEditor/components/FeedbackLog/FeedbackLog.tsx

```diff
@@ -73,9 +73,9 @@ export const FeedbackLog: React.FC<FeedbackLogProps> = ({ feedback }) => {
       field: "feedbackScore",
       headerName: "Rating",
       width: 125,
+      type: ColumnFilterType.SINGLE_SELECT,
       columnOptions: {
-        filterable: false, // TODO: make filterable
-        valueFormatter: (params) => EmojiRating[params],
+        valueOptions: EmojiRating,
```

Comment:
We could try to treat this as a "number" column (allowing filter operators like >=, <, <= etc).

However, I've opted against this as we're displaying a text status to the Editors here - it makes sense to treat these as either discrete text categories or numbers, not a mixture of both.